examples/jsm/utils/TextureUtils.js

```diff
@@ -83,6 +83,7 @@ export function decompress( texture, maxTextureSize = Infinity, renderer = null
 	readableTexture.magFilter = texture.magFilter;
 	readableTexture.wrapS = texture.wrapS;
 	readableTexture.wrapT = texture.wrapT;
+	readableTexture.colorSpace = texture.colorSpace;
```

Comment:
I think it's better for consistency if the color space of the original texture is retained.